core/DependencyInjection/ContainerServiceProvider.php

```diff
@@ -24,13 +24,14 @@
 
 namespace oat\generis\model\DependencyInjection;
 
+use Relay\RelayBuilder;
```

Comment:
Btw, my PHP stop orders it alphabetically (as I see most of people has the same config). Ordering by size can cause unordered dependencies for the same package. 

Maybe we should consider order it alphabetically?

Reply:
Done